extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -89,8 +88,14 @@ public function render( $markup, Parser $parser, PPFrame $frame, $params = null
 
 		$theme = $this->getThemeWithDefault( $params, $frame );
 		$layout = $this->getLayout( $params );
+		$accentColor = $this->getColor( 'accent-color', $params, $frame );
+		$accentColorText = $this->getColor( 'accent-color-text', $params, $frame );
+
+		//TODO: pass colors to render service
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)